src/main/drivers/compass/compass_hmc5883l.c

```diff
@@ -194,31 +264,25 @@ static bool hmc5883lInit(void)
     bool bret = true;           // Error indicator
 
     delay(50);
-#ifdef USE_MAG_SPI_HMC5883
-    hmc5883SpiWriteCommand(HMC58X3_R_CONFA, 0x010 + HMC_POS_BIAS);   // Reg A DOR = 0x010 + MS1, MS0 set to pos bias
-#else
-    i2cWrite(MAG_I2C_INSTANCE, MAG_ADDRESS, HMC58X3_R_CONFA, 0x010 + HMC_POS_BIAS);   // Reg A DOR = 0x010 + MS1, MS0 set to pos bias
-#endif
+
+    hmc5883lWriteRegister(HMC58X3_R_CONFA, 0x010 + HMC_POS_BIAS);   // Reg A DOR = 0x010 + MS1, MS0 set to pos bias
```

Comment:
`0x10 | HMC_POS_BIAS`
Also
`#define HMC_CONFA_ODR_15HZ 0x10` and `HMC_CONFA_ODR_15HZ | HMC_CONFA_POS_BIAS`

Reply:
Removed all magic numbers with proper definitions.